dags/burnham.py

```diff
@@ -202,35 +204,17 @@
 WANT_TEST_SPACE_SHIP_READY_PING = [{"count_documents": 3}]
 
 # Sensor templates for the different tables
-DISCOVERY_SENSOR_TEMPLATE = """
+SENSOR_TEMPLATE = """
 SELECT
-  COUNT(*) >= 10
+  COUNT(*) >= {min_count_rows}
 FROM
-  `{project_id}.burnham_live.discovery_v1`
+  `{project_id}.burnham_live.{table}`
```

Comment:
I like this refactor of making table configurable 👍

Reply:
Thank you! 😄